syntax = "proto2";
import "response.proto";

message Response_GetAdminNotes {
    extend Response {
        optional Response_GetAdminNotes ext = 1018;
    }
    optional string user_name = 1;
    optional string notes = 2;
}
